invenio_administration/assets/semantic-ui/js/invenio_administration/src/components/Formatter.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+import React from "react";
+import PropTypes from "prop-types";
+import _get from "lodash/get";
+import DateFormatter from "./DateFormatter";
+
+const elementTypeMap = {
+  datetime: DateFormatter,
+  date: DateFormatter,
+};
+
+class Formatter extends React.Component {
+  render() {
+    const { resourceSchema, result, property } = this.props;
+    const resourceSchemaProperty = property.replace(/\./g, ".properties.properties.");
```

Comment:
why do we need this parsing ?

Reply:
We need this mainly because the `resourceSchema` has the `properties` attribute that is auto-generated but the user is/should not be aware, the user input config is the following:
```
item_field_list = {
        "metadata.title": {"text": "Title", "order": 1},
        "slug": {"text": "Slug", "order": 2},
        "updated": {"text": "Updated", "order": 4},
        "metadata.type.id": {"text": "Type", "order": 3}
    }
```
To be able to retrieve the type of any of the property we need to parse the `metadata.title` to match the path in our `resourceSchema` to get the type.

P.S. I updated the parsing as the `resourceSchema` was updated.